src/debug.rs

```diff
@@ -47,9 +47,11 @@ macro_rules! debug {
 macro_rules! debug {
 
     ($fmt:literal) => {
-        println!("{}", format!($fmt));
+        #[cfg(debug_assertions)]
```

Comment:
They are same with non-simulator configuration. The macro with "#[cfg(feature = "native-simulator")]" can be removed.